### DISCUSSION TITLE
resize faied for scratchpad

### DISCUSSION THREAD

**Discussion by OriLiMu** [i3/i3#5760] [Q&A] [answered]
My settings: 

```
for_window [title="__scratchpad"] move scratchpad
```

I start alacritty with command `alacritty --title __scratchpad`

Then I run command 
`i3-msg "scratchpad show, resize set 800 600"`. 

The alacritty shows with very small size. 

How to fix it?

**Comment by okraits** [comment, marked as answer]
`i3-msg "scratchpad show, resize set 800 600"` works here and shows the window in the scratchpad with the size 800x600 pixels, as specified.

Did you try changing the pixel values and see if that corresponds to the size you get?
